documentation/src/main/resources/_posts/2021-09-16-conditional-requests.md

```diff
@@ -0,0 +1,224 @@
+---
+title: "Support conditional requests for things resources"
+published: true
+permalink: 2021-09-16-conditional-requests.html
```

Comment:
Could we re-date the blogpost to 22. or 23.09. ?
Also the filename. :)

Reply:
Yes - this is already on my To-Do list ;)